pages/eligibility/index.tsx

```diff
@@ -71,8 +58,9 @@ const Eligibility: NextPage<ResponseSuccess | ResponseError> = (props) => {
             className={({ selected }) =>
               selected
                 ? 'bg-white font-semibold p-2.5 pt-1.5 border border-t-4 border-content/90 border-r-muted/20 border-b-muted/20  border-l-muted/20 mr-2'
-                : 'bg-[#EBF2FC] font-semibold p-2.5 border border-muted/20 disabled mr-2'
+                : 'bg-[#EBF2FC] font-semibold p-2.5 border border-muted/20 disabled mr-2 disabled:cursor-not-allowed disabled:bg-muted disabled:text-white'
```

Comment:
The disabled Results/Estimate coloring is too dark imo, a lighter touch would appear more "disabled" to me. But, if you and Alla are cool with the color, then that's fine with me. It just looks dark, and not disabled, to me.

Reply:
I think there might be a disabled style I can yoink, but it might fail accessibility requirements the way it is. I can change the text colour to white and update the grey we use.

Reply:
Yeah, it's a bit better I think. We'll see what the team says.